src/grub.cfg

```diff
@@ -49,9 +49,9 @@ function load_video {
   fi
 }
 
-serial --speed=115200
-terminal_input serial console
-terminal_output serial console
+# Any non-default console settings will be substituted here.
+# CONSOLE-SETTINGS
```

Comment:
optional: might be worth linking to this PR here (I know git-blame works, but sometimes code gets moved around).

Reply:
Eh, this is potentially a user-facing config file, and `CONSOLE-SETTINGS` is greppable.